src/user_config.rs

```diff
@@ -249,6 +250,7 @@ impl UserConfig {
         copy_album_url: Key::Char('C'),
         audio_analysis: Key::Char('v'),
         basic_view: Key::Char('B'),
+        add_item_to_queue: Key::Char('z'),
```

Comment:
I think you'll also want to add `add_item_to_queue` in a few other places in this file. As an example, look at the `basic_view` prop.

Reply:
I believe I have fix this in eb6416d